roles/satellite-clone/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # more info about these variables can be found in satellite-clone-vars.sample.yml in the root directory of this project
-satellite_version: 6.2
+satellite_version: 6.1
```

Comment:
My reasoning behind asking to remove this variable from defaults file:
1. All the variables in defaults file are also in satellite-clone-vars-sample.yml. But after this PR, satellite_version will remain in defaults only and not in satellite-clone-vars-sample.yml - this confuses me as a user of satellite-clone.  It is not intuitive for me if you are asking me to override this value or not.
2.  There is no need to expose a variable in defaults if the value of that variable is directly derived in the code. Inspite of user passing this variable or not, you are going to determine its value in the code anyway.  This is something similar to a local variable which is defined and used in the code only and not exposed.